### PR TITLE
Fix the native compilation of Ltac2 in hybrid mode.

### PR DESCRIPTION
**Pull request by ppedrot**
Let's see if this works. cc @ejgallego

### CODE REVIEW

**Comment by ppedrot** [comment]
It works, but it spits out a flurry of warnings about `_build_vo/default//lib/coq/plugins`. I believe this directory shouldn't be added to the loadpath anyways since it doesn't contain vo files anymore (right?). Let's remove it for the time being, waiting for the loadpath code factorization.

**Comment by ppedrot** [comment]
CI green and problem fixed, this is good to go.

File: topbin/coqnative_bin.ml [hunk 1]
```diff
@@ -236,12 +236,9 @@ let init_load_path includes =
   let user_contrib = coqlib/"user-contrib" in
   let xdg_dirs = Envars.xdg_dirs in
   let coqpath = Envars.coqpath in
-  let plugins = coqlib/"plugins" in
   (* NOTE: These directories are searched from last to first *)
   (* first standard library *)
   add_rec_path ~unix_path:(coqlib/"theories") ~coq_root:(Names.DirPath.make[coq_root]);
-  (* then plugins *)
-  add_rec_path ~unix_path:plugins ~coq_root:(Names.DirPath.make [coq_root]);
```
Comment (ejgallego) [inline topbin/coqnative_bin.ml:244]: I see, this code got outdated due to the other PRs (in concrete the one doing the build layout) ; you can still keep the `add_rec_path` if you update the above line to be `coqcorelib/"plugins"`.

**Review by ejgallego — APPROVED** [review]
LGTM, bug was due to non detected conflict with the other build prs.

File: topbin/coqnative_bin.ml (hunk 1 above)
Comment (ppedrot) [reply topbin/coqnative_bin.ml:244]: The native compiler only needs access to the `vo` files, so there is little point in adding the `plugins` directory to the path.

**Comment by ejgallego** [comment]
@coqbot: merge now

**Pull request merged**